### PR TITLE
Delete student

### PR DESCRIPTION
**Pull request by davidabrahams**

### CODE REVIEW

**Comment by davidabrahams** [comment]
#182 sort of addresses this, since it would allow Anja and Bianca to do it

File: public/javascripts/studentControllers.js [hunk 1]
```diff
@@ -1,8 +1,21 @@
 var studentController = function($scope, $http, $state) {
 
-  $scope.$state = $state;
+  // $scope.$state = $state;
```
Comment (nmohamed) [inline public/javascripts/studentControllers.js:3]: delete?

**Comment by nmohamed** [comment]
:palm_tree: :bikini: :sun_with_face: :watermelon: :sweat_drops:

**Comment by aconsilvio** [comment]
It is more normal to have the delete button on the left or right? I don't want them to accidentally hit i all the time.  I think it might make more sense to have submit on the left and delete on the right - thoughts?

Otherwise 🚌

**Comment by davidabrahams** [comment]
I'm fine either way feel free to change

**Comment by BrennaManning** [comment]
:+1:

**Comment by BrennaManning** [comment]
merging!

**Pull request merged**